src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -49,22 +61,59 @@ public PersonCard(ReadOnlyPerson person, int displayedIndex) {
     }
 
     /**
+     * Get a color for a tag
+     */
+    private static String colorGetterForTag(String tagValue) {
```

Comment:
A method should be named as a verb, not a noun. This name sounds like a class rather than a method.